test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -292,10 +292,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.observability.unsafe.alertDetails.observability.enabled (boolean)',
         'xpack.observability.unsafe.thresholdRule.enabled (boolean)',
         'xpack.observability_onboarding.ui.enabled (boolean)',
-        /**
-         * xpack.discoverLogExplorer.featureFlags is conditional and will never resolve if used in non-serverless environment
-         */
-        'xpack.discoverLogExplorer.featureFlags.deepLinkVisible (any)',
+        'xpack.observabilityLogExplorer.navigation.showAppLink (any)',
```

Comment:
**nit:** should this be of type `(boolean)`?

Reply:
I tried. The test expects `any` - possibly because it loses the type due to the serverless guard.

Reply:
@weltenwort Could you add a short comment to explain the `any` type? I know we don't have them for all of the `any`-typed items here, but we're trying to be better about this.